### ISSUE TITLE
Could you share the code for training the deep model?

### ISSUE THREAD

**Issue by huangmozhilv** [cojocchen/covid19_critically_ill#2]
Thank you very much for the kindness to share the code. However, I'm very interested in the deep survival model you created and how it was trained. Is it possible to share the code for the deep survival model, including the neural network, loss function, and the code used for training?

Thank you again.

**Comment by cojocchen** [comment]
Sorry we do not have any plan releasing the training code at this moment. For details of deep survival model, I recommend the paper: "DeepSurv: personalized treatment recommender system using a Cox proportional hazards deep neural network". You can find different implementations on github by searching DeepSurv.

**Comment by huangmozhilv** [comment]
> Sorry we do not have any plan releasing the training code at this moment. For details of deep survival model, I recommend the paper: "DeepSurv: personalized treatment recommender system using a Cox proportional hazards deep neural network". You can find different implementations on github by searching DeepSurv.

Okay~ Thanks a lot.
